### PR TITLE
Honour the 'tests' field on input request correctly.

### PR DESCRIPTION
**Pull request by peterebden**
Also better handle the case of no input files.

### CODE REVIEW

**Review by Tatskaari — APPROVED** [review]
Though I don't see us passing the tests flag?

File: tools/driver/ChangeLog [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+    * Better handle the case of no input files
+    * Honour the 'tests' field on input request correctly.
+
 Version 0.2.1
 -------------
     * `go_src` dependencies are now explicitly built, ensuring that e.g. code generated from
       protobufs gets downloaded when using remote execution.
 
-Version 0.2.0
+ersion 0.2.0
```
Comment (Tatskaari) [inline tools/driver/ChangeLog:11]: Typo

**Comment by peterebden** [comment]
> Though I don't see us passing the tests flag?

It comes in on the request, which is sent on stdin; the caller can choose whether they want it or not.

File: tools/driver/ChangeLog (hunk 1 above)
Comment (peterebden) [reply tools/driver/ChangeLog]: Oops

**Pull request merged**